sts-mock/src/token/validateServiceTokenRequest/validateServiceTokenRequest.ts

```diff
@@ -0,0 +1,40 @@
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export type IValidateServiceTokenRequest = (
+  requestBody: string | null,
+) => Result<ValidServiceTokenRequestParams>;
+
+export type ValidServiceTokenRequestParams = {
+  subjectId: string;
+  scope: string;
+};
+
+export const validateServiceTokenRequest: IValidateServiceTokenRequest = (
+  requestBody,
+) => {
+  if (requestBody == null) {
+    return errorResult({
+      errorMessage: "Missing request body",
+      errorCategory: "CLIENT_ERROR",
+    });
+  }
+  const searchParams = new URLSearchParams(requestBody);
+
+  const subjectToken = searchParams.get("subject_token");
+  if (!subjectToken) {
+    return errorResult({
+      errorMessage: "Missing subject_token",
+      errorCategory: "CLIENT_ERROR",
+    });
+  }
+
+  const scope = searchParams.get("scope");
+  if (!scope) {
```

Comment:
I'm wondering if we should also check the scope value. This would guard against the possibility of us accidentally changing it in one of our lambdas?